### DISCUSSION TITLE
where i can find the source code for the floating-edges demo

### DISCUSSION THREAD

**Discussion by ulissepress** [bcakmakoglu/vue-flow#46] [Q&A] [answered]
Sorry but looking at the github repo I am not able to find the source code for this demo:
[https://vueflow.dev/examples/floating-edges](https://vueflow.dev/examples/floating-edges)

**Comment by bcakmakoglu** [comment, marked as answer]
For 0.3.x you can check the source of the docs right [here](https://github.com/bcakmakoglu/vue-flow-docs/blob/develop/pages/examples/floating-edges.vue)

[Look here for 0.4.0-x](https://github.com/bcakmakoglu/vue-flow-docs/blob/feat/vue-flow-v0.4.0/pages/examples/floating-edges.vue)

**Reply by ulissepress** [reply]
thank you!

**Reply by Xsmael** [reply]
What happened to floating edges ?  I can't see it in the docs or examples and all links reated to it show 404 Page 
I really needed this feature, but i can't find it no where not even on the git repo.

Thanks

**Reply by bcakmakoglu** [reply]
You can still find the floating edges example right [here](https://github.com/bcakmakoglu/vue-flow/tree/master/examples/vite/src/FloatingEdges).

**Reply by Xsmael** [reply]
Got it, thanks!   
Just to satisfy my curiosity, is there any particular reason why it's not in the official docs examples with other examples ?

**Reply by bcakmakoglu** [reply]
Back when it was on the docs page I ran into a little bit of licensing trouble with React Flow about it so I took it down from the docs 😉
I don't think it's an issue anymore but if I put it back up, I'll probably modify it enough as to not be an exact copy of their code.

**Reply by Xsmael** [reply]
Oh alright, thanks for the info now i get it.  (Sorry, I have this issue that it iches me when i don"t understand something 😅  so i had to ask)
